oceanmodes/test/test_modes.py

```diff
@@ -5,8 +5,14 @@
 import numpy as np
 import unittest
 import oceanmodes as modes
+reload(modes)
```

Comment:
You don't want to call reload from within the test script. This is also why the tests fail for python3.